### ISSUE TITLE
Unable to Locate Chromium

### ISSUE THREAD

**Issue by ricardodomingues10** [Lissy93/web-check#108]
I tried this command: docker run -p 3000:3000 lissy93/web-check, and encountered this problem.

Error details tech-stack:
Could not find Chromium (rev. 1095492). This can occur if either:
1. you did not perform an installation before running the script (e.g. `npm install`) or
2. your cache path is incorrectly configured (which is: /root/.cache/puppeteer).
For (2), check out our guide on configuring puppeteer at https://pptr.dev/guides/configuration.

Can you help me?

**Comment by m4teh** [comment]
Defining CHROME_PATH=/usr/bin/chromium also doesn't work. Despite this being the location in the [published docker image](https://hub.docker.com/r/lissy93/web-check).

**Comment by Error00101** [comment]
Had the same issue.
@m4teh 
Defining the 'CHROME_PATH' does not work as it is already defined inside the container (just run env inside the container -> you will see it is defined)

however I dug through some of the sourcecode. ( -> API -> tech-stack -> wappalyzer(v6.10) ->puppeteer)
puppeteer has some documentation where I found a few Interesting thing:
PUPPETEER_CACHE_DIR -> nope
PUPPETEER_EXECUTABLE_PATH -> YES this is it (Somewhere through the chain the CHROME_PATH gets lost and now replaced with this - didnt look where but semms to be solvable)

<details>

<summary>Screenshots of it not working/Working with the coresponding Compose files</summary>
not working :(

![grafik](https://github.com/user-attachments/assets/95735c24-18d4-444f-a375-a2bfb98c54d0)

now working :)
![grafik](https://github.com/user-attachments/assets/a9df5a2f-65d5-456c-b63c-bee433e08fae)

</details>

TLDR:
Set the env -> PUPPETEER_EXECUTABLE_PATH=/usr/bin/chromium


If i get my hands on some time I will debug this.

**Comment by b1nary-b0b** [comment]
Thx for thw workaround, unfortunately it does not work for me :( I still get the same error

**Comment by maxinux** [comment]
Still was broken for me on the docker image, puppeteer_executable_path fixed it though

**Comment by lissy93** [comment]
Should be fixed from the May release, sorry - forgot to update this ticket. Lemme know if still having issues.
